install.md

```diff
@@ -95,6 +95,24 @@ now?</q>. Confirm this by clicking <q>Install</q>.</li>
 contains executables such as <code>clang</code> and <code>clang++</code>,
 the C and C++ compiler.</li>
 </ol>
+<p>
+Note: some users have experienced build failures on macOS 10.15 with Xcode 11.4
+(cf. this <a href="https://github.com/oscar-system/Oscar.jl/issues/82">github issue</a>).
+Until a proper fix is found, the following might serve as a workaround, consisting in building <code>libcxxwrap-julia</code>
+with Xcode 11.4 instead of using the binaries (replace <code>$SOMEWHERE</code> by
+the directory in which you wish to install <code>libcxxwrap-julia</code>):
```

Comment:
I don't think the average user will have a clue what directory they "wish" to install this. We should make a suggestion to them, as concrete as possible. For example, how about `$HOME/.julia/cxxwrap-workaround/`